src/components/WebexRemoteMedia/WebexRemoteMedia.js

```diff
@@ -40,7 +64,13 @@ export default function WebexRemoteMedia({className, meetingID}) {
               <div>Connecting</div>
             </Badge>
           ) : null}
-          {remoteVideo ? <video ref={videoRef} playsInline autoPlay /> : null}
+
+          {remoteVideo ? <video ref={videoRef} playsInline autoPlay name="remote-cam" /> : null}
```

Comment:
I don't think [`name` is an attribute of `video`](https://developer.mozilla.org/en-US/docs/Web/API/Element/name) elements

```suggestion
          {remoteVideo ? <video ref={videoRef} playsInline autoPlay /> : null}
```

Reply:
This custom attribute was used to differentiate the two video element when they appeared at the same time. Removing them is fine.